gcp/opsverse-gke/vars.tfvars

```diff
@@ -17,97 +27,97 @@ node_locations      = "us-east4-a,us-east4-b,us-east4-c"
 # 3 Public Subnets and 3 Private Subnets by default
 # This can be changed as per requirements
 subnets = [
+  {
+    subnet_name   = "opsverse-gke-cluster-apps-public-a"
```

Comment:
What is the change here? It becomes hard for reviewers to understand the change/diff. 
Please avoid mixing the lint changes with functional changes going forward.

Reply:
There are no functional changes here, only **terraform fmt** was applied to format the .tf files according to standard guidelines .In the future, I'll make sure to distinguish formatting modifications from functional upgrades.